assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/AudienceTiles.js

```diff
@@ -508,6 +511,16 @@ export default function AudienceTiles( { Widget, widgetLoading } ) {
 						/>
 					);
 				} ) }
+				{ ! isTabbedBreakpoint && visibleAudiences.length === 1 && (
+					<Fragment>
+						{ loading && (
+							<Widget noPadding>
+								<AudienceTileLoading />
```

Comment:
Although it was missed during definition, we need to show a tile in the loading state before showing the placeholder tile.

Reply:
Nice one. Thanks, @techanvil 👍